services/dynamic-scheduler/tests/unit/api_rest/test_api_rest__health.py

```diff
@@ -11,7 +11,9 @@
 
 @pytest.fixture
 def app_environment(
-    disable_rabbitmq_setup: None, app_environment: EnvVarsDict
+    disable_services_tracker_setup: None,
+    disable_rabbitmq_setup: None,
+    app_environment: EnvVarsDict,
```

Comment:
question: do you actually have a real healthcheck or it only always returns 200?
in case redis is kaputt? or in case rabbitmq is kaputt?